docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+ - accesibility of your dataset in the major single cell data formats (including `AnnData`, `seurat`, and `loom`)
+
+This tutorial will consist of an explanation of 1) how to create and structure an `AnnData` object with your single cell data 2) how to augment this object with information that is specific to the cellxgene schema and 3) how to upload this object to the cellxgene data portal. If you run into any issues during this tutorial, or have any suggestions on how to improve the portal and curation experience, you can contact us via [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+<!---
+You can also check out the following links for more information on the cellxgene ecosytem:
+
+- [cellxgene data portal](https://cellxgene.cziscience.com/)
+- [cellxgene explorer github](https://github.com/chanzuckerberg/cellxgene)
+- [cellxgene data portal github](https://github.com/chanzuckerberg/corpora-data-portal)
+- [corpora schema](https://github.com/chanzuckerberg/single-cell-curation/blob/main/docs/corpora_schema.md)
+- [schema guide](https://github.com/chanzuckerberg/single-cell-curation/blob/main/docs/schema_guide.md)
+
+--->
+
+### Table of Contents
+
+- [Quick start](#quick-start)
+- [Data requirements and AnnData structure](#required-data-and-anndata-structure)
+  - [Format conversion](#format-conversion)
+  - [Alternative assays](#alternative-assays) 
+- [Schema definition](#schema-definition)
+  - [Basic requirements](#basic-requirements-expanded-version)
+  - [`uns`](#uns)
+  - [`obs`](#obs) 
+- [Cellxgene curation tools](#cellxgene-curation-tools)
+  - [Installation](#installation)
+  - [`config.yaml`](#configyaml)
+    - [`uns` section](#uns-section)
+    - [`obs` section](#obs-section)
+    - [`fixup_gene_symbols` section](#fixup_gene_symbols-section-gene-symbol-harmonization)
+  - [`cellxgene schema apply`](#cellxgene-schema-apply)
+  - [`cellxgene schema validate`](#cellxgene-schema-validate)
+  - [Local testing and dropbox upload](#testing-locally-optional-and-upload-to-dropbox)
+- [Uploading data to the cellxgene data portal](#uploading-data-to-the-cellxgene-data-portal)
+  - [Portal sign in](#portal-sign-in)
+  - [Create a private collection](#create-a-private-collection)
+  - [Add a dataset](#add-a-dataset-to-a-private-collection)
+  - [Remove a dataset](#remove-dataset-from-a-private-collection)
+  - [Dataset actions](#dataset-actions)
+  - [Publish collection](publish-collection-to-the-portal)
+- [Gene sets (under construction)](#gene-sets)
+
+<br/>
+
+---
+
+## Quick start
+
+<br/>
+
+If you are already familiar with cellxgene, AnnData, and the cellxgene data schema, then give curation a shot with this [quick start guide](https://github.com/chanzuckerberg/single-cell-curation#quick-start). Otherwise, continue reading!
+
+---
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/111403110-d1589700-86a2-11eb-99dd-bf52e348d43a.png) --->
+
+<!---
```

Comment:
Please eliminate commented out sections. It's confusing in markdown reviews. If you're saving for some future reason, then cut-n-paste into a separate markdown document.